### ISSUE TITLE
`load_conllu` throws errors with enhanced dependencies in UD-r2.2

### ISSUE THREAD

**Issue by venkatasg** [hltcoe/PredPatt#14]
In UD-r2.2, to accomodate the use of empty nodes for the analysis of ellipsis in enhanced dependencies, the HEAD(gov in code) column is set to _. This throws an error in the `load_conllu` function, since `DepTriple` is called with `int(gov)` as one of the arguments. UD explains these nodes [here](http://universaldependencies.org/format.html#words-tokens-and-empty-nodes) and [here](http://universaldependencies.org/u/overview/enhanced-syntax.html).

Fix is easy enough. One can check the first column for '.', since UD stipulates that empty nodes must have index of the form i.1, where i is the index of referent of ellipsis. If '.' exists, ignore that line. Unless there is some information we can extract from the empty node?

**Comment by sheng-z** [comment]
Yes, that's an easy workaround, but it loses empty nodes which are potentially useful for PredPatt. A better solution is to rewrite the way PredPatt deals with the UD index. That requires more efforts.

**Comment by venkatasg** [comment]
I’m not sure how useful it will be for PredPatt. Firstly I could find no more than 30 instances of this kind of empty nodes in UD. I’ll look into specific examples but in a sentence like ‘Bill ate cookies, and Tom cake’, the HEAD of Tom is still ate I believe? Or perhaps conjunction plays a role in this.

**Comment by venkatasg** [comment]
Does PredPatt use any information from enhanced dependencies at all? Right now I don't think so, and if we want it to in future, that will involve a lot of changes. For now, I think its a good idea to just ignore empty nodes, so that PredPatt doesn't throw an error with UD-r2.2.
